pages/about/authorGuidelines.md

```diff
@@ -0,0 +1,83 @@
+
+When the equation is at the end of an item, end the definition with a comma, follow with an equal sign and  the rest of the equation, then end with a period as shown in the following example.
+
+Consecutive Numbering: Equations within an article are numbered consecutively from the beginning of the  article to the end.
+Appendix Equations: Continued consecutive numbering of equations is best in the Appendix, but equation  numbering that starts over with (A1), (A2), etc., for Appendix equations is permissible.
+
+ Material in displayed equations is automatically italic unless you indicate otherwise. Some simple general rules apply. All variables are italic. Function names and abbreviations are Roman, as are units, unit abbreviations, complete words, and abbreviations of words. Superscripts and subscripts follow this same formula: when they are variables, they are italic; when they are abbreviations of words (such as “in” and “out” for input and output), they are Roman. Single-letter superscripts and subscripts may be italic even if they are abbreviations, unless this leads to inconsistency between italic and Roman characters for similar types of subscripts.
+
+### References
+
+Please consult the decicated section on references and other types of Papers in {% cite EditorialStyleManualAuthors %}.
+
+### Text citation of Figures and Tables
+
+Please consult the decicated section on text citations of Figures and Tables in {% cite EditorialStyleManualAuthors %}.
+
+### Other text
+
+Please consult the decicated section on inclusive language, footnotes, and lists in text in {% cite EditorialStyleManualAuthors %}.
+
+### Writing style
+
+Please consult the decicated section on : Acronyms, spelling, trademarks, plurals, hyphenation rules, grammar, contractions, capitalization, Dates, percentages and decimals, Ranges wit units in chapter E. Writing Style for Transactions in {% cite EditorialStyleManualAuthors %}.
+
+### Grammar and usage
+
+For guidance in grammar and usage, please consult the  {% cite Chicago:ManualOfStyle %} and Chapter III. Grammer and usage in transactions from {% cite EditorialStyleManualAuthors %}.
+
+### Mathematical editing
+
+please consult the {% cite EditingMathematics %} for guidelines on how to write mathematics.
+
+### Electrical and Electronics Terms, notations and conventions
+
+please consult the {% cite StandardDictionary %} for the terms to be used in Electrical and Electronics fields.
+
+For notations and conventions, please look at this [page](pages/about/notationsAndConventions.md)
+
+For electrical and eletronic diagram, you can refer to {% cite GraphicSymbolsElectricalElectronics %} and {% cite SupplementGraphicSymbolElectricalElectronics %}
+
+For control terminology, you can check: {% cite AutomaticControlSystemDictionary %}
+
+## References
+
+{% bibliography --file authorGuidelines  %}
```

Comment:
Still the same issue with reference 7:
![image](https://github.com/CRESYM/colib0.github.io/assets/157120809/70109d89-38ca-405b-bac7-e00d2d7938fc)